src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -188,7 +188,8 @@ public function findByIds($ids)
             ->addSelect('faxType')
             ->addSelect('notes')
             ->addSelect('medias')
-            ->where('account.id IN (:accountIds)');
+            ->where('account.id IN (:accountIds)')
+            ->orderBy('account.id', 'ASC');
```

Comment:
Would it be a lot of work to fix the tests instead of adding an order here? That actually has a performance impact... Same for all the other orders in here.